Source/WebCore/layout/formattingContexts/grid/TrackSizingAlgorithm.cpp

```diff
@@ -31,8 +31,14 @@
 namespace WebCore {
 namespace Layout {
 
+struct UnsizedTrack {
+    LayoutUnit baseSize;
+    LayoutUnit growthLimit;
+    const TrackSizingFunctions trackSizingFunction;
```

Comment:
I'm surprised this struct and `UsedTrackSizes` can be forward declared and only defined in these implementation files! I wonder if we should (eventually) make another `*Aliases.h` header at some point for these types. Surely someone in another file will eventually want access a member of this structure? (But perhaps not...)

Reply:
I guess these were private definitions, so maybe it's totally fine for them to be in the implementation file after all.